src/components/Heroes/columns.jsx

```diff
@@ -23,72 +22,72 @@ export default {
     displayName: strings.hero_pick_ban_rate,
     field: 'pickBanRatePro',
     sortFn: true,
-    displayFn: (row, col, field) => percentileDisplay(field, row.matchCountPro),
+    percentBars: row => decimalToCount(row.pickBanRatePro, row.matchCountPro),
```

Comment:
shouldn't percentbars take row.pickBanRatePro since it expects a decimal?  Or just make this sortFn and then pass `true`?